configs/default/HNEC-NIDICI_F4.config

```diff
@@ -0,0 +1,110 @@
+# Betaflight / STM32F405 (S405) 4.2.5 Nov 22 2020 / 18:38:23 (afdac08b3) MSP API: 1.43
+
+board_name NIDICI_F4
+manufacturer_id HNEC
+
+# resources
+resource BEEPER 1 B04
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 C09
+resource MOTOR 4 C08
+resource PPM 1 A03
+resource LED_STRIP 1 B06
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 6 C07
+resource INVERTER 2 C13
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 B05
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 C12
+resource CAMERA_CONTROL 1 B07
+resource ADC_BATT 1 C02
+resource ADC_CURR 1 C01
+resource PINIO 1 C13
+resource FLASH_CS 1 A15
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 C05
+
+# timer
+timer A03 AF1
+# pin A03: TIM2 CH4 (AF1)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer B09 AF3
+# pin B09: TIM11 CH1 (AF3)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature TELEMETRY
+feature OSD
+
+# serial
+serial 1 64 115200 57600 0 115200
+
+# master
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set current_meter = ADC
+set battery_meter = ADC
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set dashboard_i2c_bus = 1
+set pinio_box = 40,41,255,255
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+
```

Comment:
I'm not sure why you removed the gyro alignment. It is required that you specify the alignment of the sensor. You should make sure that you select the correct alignment so that the motion of the 3D model on the Setup tab matches movements of the board when the flight controller is orientated in the expected installation. Please test your changes before submitting more updates to the target.

Reply:
Maybe I misunderstood comments in PR#430 related with board alignments. Now I restored the "set gyro_1_sensor_align = CW180FLIP" line which is necessary for gyro sensor1 align with the Indicator printed on the board. Our engineers have tested this config file works fine .  Thank you for your guidance !